packages/material-ui-lab/src/Autocomplete/Autocomplete.js

```diff
@@ -25,13 +25,23 @@ export const styles = theme => ({
   /* Styles applied to the tag elements, e.g. the chips. */
   tag: {
     margin: 3,
+    maxWidth: 'calc(100% - 6px)',
```

Comment:
What's wrong with?

```suggestion
    maxWidth: '100%',
```

Reply:
We need to take the margin into consideration as it's not being included in the width. Otherwise it still will make the Chip bigger than it should

Reply:
Ok thanks